### ISSUE TITLE
Trace was Paid out but status didnt change?

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapp#2150]
https://beta.giveth.io/milestone/jeffs-june-20-work

weird that it says completed... but paid out here: https://etherscan.io/tx/0x2d6b255e815723247a387e95efdcb6e08112c67aa06d3cd8c0d134e900f0239c

**Comment by aminlatifi** [comment]
@mohammadranjbarz You are right, there is an issue in event processing. The `Transfer` events corresponding to withdraw transaction are all in `Pending` state!!

{transactionHash:"0xf00b6e53e61e08f3daf0edfa39798e29c9d1541d5a8d33d643d0e3bb8b2bd6cb", event:"Transfer"}
